packages/styles/src/mixins/_button.scss

```diff
@@ -34,14 +35,11 @@
   }
 }
 
-@mixin icon-button($size: md) {
-  // Padding for icon buttons is (button-height - font-size) / 2
-  $icon-size: map.get(button.$btn-icon-size-map, $size);
-  $button-height: map.get(button.$btn-height-map, $size);
-  $padding-y: ($button-height - $icon-size) * 0.5 - button.$input-btn-focus-width;
-  min-width: $button-height;
-  padding-right: $padding-y;
-  padding-left: $padding-y;
+@mixin button-variant-def($type, $color, $inverted: false) {
+  $inverted-suffix: if($inverted, '-inverted', '');
+  border-color: tokens.get('button-#{$color}#{$inverted-suffix}-#{$type}-stroke');
+  background-color: tokens.get('button-#{$color}-#{$type}-bg');
+  color: tokens.get('button-#{$color}-#{$type}-fg');
```

Comment:
The inverted tokens should not be used (Alessio will remove them). The schemes token layer will handle the correct mapping in light and dark mode.